Page for the 2023 MFEM community workshop

tzanio opened the pull request:
<img width="974" alt="Screenshot 2023-07-10 at 10 33 50 AM" src="https://github.com/mfem/web/assets/1278247/dce48c17-50a4-438b-bf7e-83ea4bfe169f">

kmittal2 commented on src/workshop22.md:
```diff
@@ -0,0 +1,151 @@
+
+### Overview
+
+The MFEM team is happy to announce the second MFEM Community Workshop, which will
```
```suggestion
The MFEM team is happy to announce the third MFEM Community Workshop, which will
```

kmittal2 commented on src/workshop22.md:
```diff
@@ -0,0 +1,151 @@
+### Overview
+
+The MFEM team is happy to announce the second MFEM Community Workshop, which will
+take place on October 25, 2022, virtually, using Zoom for videoconferencing.
```
```suggestion
take place on October 25, 2022, virtually, using Zoom for video conferencing.
```

kmittal2 commented on src/workshop.md:
```diff
@@ -49,25 +42,25 @@ Workshop participants are encouraged to join the
+| 8:00-8:20 | **Welcome & Overview** | **TBD** (LLNL) |
+| 8:20-8:40 | **The State of MFEM** | **Tzanio Kolev** (LLNL) |
+| 8:40-9:00 | **Recent Developments** | **Veselin Dobrev** (LLNL) |
+| 8:00-9:20 | **Break** | All |
```
```suggestion
| 9:00-9:20 | **Break** | All |
```

hauten commented on src/workshop.md:
```diff
@@ -49,25 +42,25 @@ Workshop participants are encouraged to join the
+| 8:40-9:00 | **Recent Developments** | **Veselin Dobrev** (LLNL) |
+| 8:00-9:20 | **Break** | All |
+| 9:20-10:20 | **Talks, Session I** | (20 mins each) |
+| 10:20-10:40 | **Break & Group Photo**| Download a virtual background below |
```
```suggestion
| 10:20-10:40 | **Break & Group Photo** | Download a virtual background below |
```

kmittal2 reviewed (approved):
Other than some minor comments I left, this looks good to me.

hauten reviewed (approved):
Like Ketan, I made just a couple of small changes

kmittal2 commented on src/workshop.md:
```diff
@@ -23,24 +16,24 @@ directions for the project.
--->
+See also the agenda for the previous [2022](../workshop22), [2021](../workshop21) MFEM workshops.
 
 ### Meeting format
```
Perhaps we could rename this section `Agenda` and move the note about slack to the `Meeting format` section above.

tzanio commented:
I updated the workshop date and addressed the @kmittal2's and @hauten's comments (thank you!)

@pazner, @homijan, @psocratis and @acfisher -- do you have any additional suggestions?

tzanio commented:
Thanks for all the help and the reviews @kmittal2, @pazner, @acfisher, @psocratis, @hauten and @homijan 🙏 

This will be another awesome workshop 🚀
